pulpcore.fc

```diff
@@ -21,5 +21,6 @@
 /var/lib/pulp/upload(/.*)?	gen_context(system_u:object_r:pulpcore_var_lib_t,s0)
 /var/lib/pulp/sign-metadata.sh	--	gen_context(system_u:object_r:pulpcore_var_lib_t,s0)
 
+/var/run/pulpcore-api(/.*)?		gen_context(system_u:object_r:pulpcore_server_var_run_t,s0)
 /var/run/pulpcore-content(/.*)?		gen_context(system_u:object_r:pulpcore_server_var_run_t,s0)
```

Comment:
Using this as a way to learn more about SELinux: is separate lines a best practice or would you combine them in the regex?

```suggestion
/var/run/pulpcore-(api|content)(/.*)?		gen_context(system_u:object_r:pulpcore_server_var_run_t,s0)
```

Reply:
This join looks reasonable. Generally the rules are readability or performance, which is hard to measure though. Surely regexps are harder to grep in e. g. `semanage fcontext -l` command output.

Reply:
Makes sense. I feel those are just general rules when it comes to regexes.